### PR TITLE
FP value support in python API

### PR DESCRIPTION
**Pull request by yoni206**
This PR adds new `is*` functions from the cpp API to the python API.
In particular, it adds `getFloatingPointValue()` function from the cpp API.
A test (translated from `term_black.cpp`) is added.

`getFloatingPointValue()` returns a tuple, and so this requires importing an instance of tuples into cython.

### CODE REVIEW

File: src/api/python/cvc5.pxd [hunk 1]
```diff
@@ -27,6 +27,29 @@ cdef extern from "<string>" namespace "std":
...
+        S third
+        tuple() except +
+        tuple(tuple&) except +
+        tuple(T&, U&, S&) except +
+        bint operator==(tuple&, tuple&)
+        bint operator!=(tuple&, tuple&)
+        bint operator<(tuple&, tuple&)
+        bint operator>(tuple&, tuple&)
+        bint operator<=(tuple&, tuple&)
+        bint operator>=(tuple&, tuple&)
```
Comment (nafur) [inline src/api/python/cvc5.pxd:46]: I think we should only "import" what we need. I think we don't need any of this (but not sure)
```suggestion
    cdef cppclass tuple[T, U, S]:
        pass
```

File: src/api/python/cvc5.pxd [hunk 2]
```diff
@@ -27,6 +27,29 @@ cdef extern from "<string>" namespace "std":
...
+
+cdef extern from "<tuple>" namespace "std":
+    uint32_t get0 "std::get<0>"(tuple[uint32_t,uint32_t,Term]) except +
+    uint32_t get1 "std::get<1>"(tuple[uint32_t,uint32_t,Term]) except +
+    Term get2 "std::get<2>"(tuple[uint32_t,uint32_t,Term]) except +
```
Comment (nafur) [inline src/api/python/cvc5.pxd:51]: I think we can properly declare them a templated functions (not 100% sure about the syntax though, see https://cython.readthedocs.io/en/latest/src/userguide/wrapping_CPlusPlus.html#templates)
```suggestion
    T get0[T,U.S] "std::get<0>"(tuple[T,U,S]) except +
    U get1[T,U.S] "std::get<1>"(tuple[T,U,S]) except +
    S get2[T,U.S] "std::get<2>"(tuple[T,U,S]) except +
```

**Review by nafur — COMMENTED** [review]
Two suggestions...

File: src/api/python/cvc5.pxd (hunk 1 above)
Comment (yoni206) [reply src/api/python/cvc5.pxd]: Right! Thanks, deleted here: f68979e6b4742f4210c2420af6fff6501061a15d

File: src/api/python/cvc5.pxd (hunk 2 above)
Comment (nafur) [reply src/api/python/cvc5.pxd:37]: Discussed offline. This does not work, and we don't know how to make it work...

**Review by nafur — APPROVED** [review]
LGTM.

**Pull request merged**